crates/nu-command/tests/commands/alias.rs

```diff
@@ -6,13 +6,7 @@ use nu_test_support::{nu, pipeline};
 #[test]
 fn alias_simple() {
     let actual = nu!(
-        cwd: "tests/fixtures/formats", pipeline(
-        r#"
-            alias bar = use sample_def.nu greet;
-            bar;
-            greet
-        "#
-    ));
+        cwd: "tests/fixtures/formats", " alias bar = use sample_def.nu greet; bar; greet ");
```

Comment:
I'm gonna revert this test for a few reasons:

1. The test is currently disabled (see the `#[ignore]`); if we accidentally break the test we won't notice
2. This test runs 3 separate commands, I think it's easier to read when they're on their own lines

Reply:
fair enough, good shout on the ignore block